Dockerfile

```diff
@@ -37,7 +37,8 @@ COPY local-registry/* ./
 RUN ${wd}/bin/generate-registry.sh
 
 # As of Dec 2019, we need to use the nightly toolchain to get JSON test output
-FROM rustlang/rust:nightly AS test
+FROM rust/nightly AS test
```

Comment:
`rust/nightly` has the same problem with caching as `rustlang/rust:nightly` does.

The idea is to use some "absolute" (as opposed to "relative") tag as a base image.
Please consider using `FROM rust:1.64`: it does not have to be a "nightly", we are adding the nightly toolchain with the very next line.

Reply:
I would also not include the `Cargo.lock`. Why changing two aspects of the someone else's system with a single PR? :)

Reply:
Sorry about that. Must've happened automatically, I didn't `cargo update` or anything